src/Pages/EventManager/EventManagerModal.js

```diff
@@ -146,15 +147,13 @@ function EventManagerModal (props) {
 
   return (
     <div>
-      <Modal isOpen={confirmationModal} toggle={toggleConfirmationModal}>
-        <ModalHeader>Delete "{props.title}"?</ModalHeader>
-        <ModalFooter>
-          <Button onClick={toggleConfirmationModal}>Cancel</Button>
-          <Button color='danger' onClick={handleDeletion}>
-            Yes, delete
-          </Button>
-        </ModalFooter>
-      </Modal>
+      <ConfirmationModal
+        headerText={`Delete ${props.title} ?`}
+        bodyText='The event will be gone forever if you do this'
```

Comment:
```suggestion
        bodyText='The event will be gone forever if you do this.'
```
![image](https://user-images.githubusercontent.com/36345325/75586562-8e8d3800-5a29-11ea-9646-50c12049f7b3.png)